Fork Public Repository Challenge

DMCatanach opened the pull request:
The "added dawn file" commit is the important change, but I had problems trying to push it to GitHub. The other two commits are the results of trying to fix those problems.

acodeinprogress commented:
Hey @DMCatanach,

If you take a look at the Files Changed tab, it says that 1,059 files were modified. That doesn't sound right to me. Do you think you could try this exercise again?

DMCatanach commented:
When I initially tried to push the commit with my added file, I ran into an issue reading from the remote repository - I think I hadn't added it. But once I did add the remote, the error message I got was "Updates were rejected because the remote contains work that you do not have locally." In trying to fix that, I ended up adding copies of the contents of the "ruby" folder. I've now deleted the duplicates, so now my forked copy is much more like what I started with, except for the dawn.rb file I added.

acodeinprogress commented:
Hey @DMCatanach,

Thanks for removing those files! Unfortunately, it still looks like a lot of extra files were edited. Good news is that you're now down to 721! :)
